README.md

```diff
@@ -45,64 +65,121 @@ Cascading Style Sheets, or CSS, is a language created to style an HTML document
   - To select multiple different elements separate them by commas like this `div, p, a`
 
 Here's an example of CSS styling:
+
 ```css
 * {
   color: red;  /* color in CSS refers to font color */
 }  /* all elements will have red font */
 ```
 
-## Making a Rainbow
-First off make sure you have [forked](https://github.com/learn-co-students/my-little-rainbow-v-000) and cloned this repo. Next, create a new branch, and switch to it; it's `git checkout -b your_solution_branch_name` in case you forgot.
+## Make a Rainbow
 
-In that directory you'll see three files. `index.html`, `main.css`, and this `README.md`.  Open them in your text editor via your command line.  Also open `index.html` in your browser; if everything is working correctly you should see a white page. Good job!
+First off make sure you have [forked](https://github.com/learn-co-students/my-
```

Comment:
YIKES. We haven't taught git yet....need to strategize about this.

Reply:
But this is basically a lab...so i think we'll need to fork this off to its own repo.